registrations/models.py

```diff
@@ -250,6 +253,69 @@ def is_user_editable_resources(self):
         )
 
 
+class RegistrationUser(models.Model):
+    email = models.EmailField(verbose_name=_("E-mail"))
+
+    registration = models.ForeignKey(
+        Registration, on_delete=models.CASCADE, related_name="registration_users"
+    )
+
+    language = models.ForeignKey(
+        Language,
+        on_delete=models.SET_NULL,
+        null=True,
+        blank=True,
+        related_name="registration_user_language",
+    )
```

Comment:
This is my personal preference for using empty lines:
```suggestion
    email = models.EmailField(verbose_name=_("E-mail"))

    registration = models.ForeignKey(
        Registration, on_delete=models.CASCADE, related_name="registration_users"
    )
    language = models.ForeignKey(
        Language,
        on_delete=models.SET_NULL,
        null=True,
        blank=True,
        related_name="registration_user_language",
    )
```
An empty line between every single field/attribute makes the code a bit more frustrating to read, especially if you have a lot of one-liners (which isn't the case here). Better to use them for grouping, e.g. regular fields, then an empty line, then all the foreign keys, etc.

For example:
```py
class SomeModel(models.Model):
    email = models.EmailField(...)
    first_name = models.CharField(...)
    last_name = models.CharField(...)

    foo = models.ForeignKey(...)
    bar = models.ForeignKey(...)

    some_custom_attribute = ...

    def method():
        ...
```
is much more pleasant to read than:

```py
class SomeModel(models.Model):
    email = models.EmailField(...)

    first_name = models.CharField(...)

    last_name = models.CharField(...)

    foo = models.ForeignKey(...)

    bar = models.ForeignKey(...)

    some_custom_attribute = ...

    def method():
        ...
```

Of course, this is completely a matter of personal preference; I just don't like having too many empty lines. Do what you will.